webots/controllers/Supervisor/supervisor.py

```diff
@@ -192,9 +165,10 @@ def main_loop():
 
                     # Robot odometry data received?
                     elif command[0] == "ODO":
-                        robot_odometry.x = int(command[1]) # [mm]
-                        robot_odometry.y = int(command[2]) # [mm]
-                        robot_odometry.yaw_angle = float(command[3]) / 1000.0 # [rad]
+                        robot_odometry.x = int(command[1])  # [mm]
```

Comment:
Is it guaranteed that `rx_data` actually contains comma separated integers?
Otherwise, the indexing with [1] might fail.

Reply:
Yes, as so is the protocol defined. This could be done differently to ensure the data is received and how many elements are found in the list, but this implementation is sufficient for the use case

Reply:
okay.